src/components/AddItem.js

```diff
@@ -30,49 +35,36 @@ const AddItem = () => {
       <fieldset>
         <legend>How soon will you buy this again?</legend>
         <div>
-          {frequency === 7 ? (
-            <input
-              onChange={e => setFrequency(parseInt(e.target.value))}
-              type="radio"
-              id="soon"
-              name="frequency"
-              value={7}
-              checked
-            />
-          ) : (
-            <input
-              onChange={e => setFrequency(parseInt(e.target.value))}
-              type="radio"
-              id="soon"
-              name="frequency"
-              value={7}
-            />
-          )}
+          <input
+            onChange={e => handleRadioChange(e)}
+            type="radio"
+            id="soon"
+            name="soon"
+            checked={frequency === 7}
```

Comment:
checked is true depending on what the value of `frequency` is.

Reply:
Oh great! Thank you!